### PR TITLE
MakeFile updated for easy electron Version Control

### PR DESCRIPTION
**Pull request by Mingling94**

### CODE REVIEW

**Comment by DavidVorick** [comment]
This will not work. You need to use variables like this:

`wget -nc https://github.com/atom/electron/releases/download/v0.26.1/$(electron-v)-linux-ia32.zip`

Additionally, I think 'electron-version' is probably a better name. Luke might disagree, but '-v' quite often means 'verbose' in the context of flags and makefiles. I'm especially for having a longer name because nobody will ever need to type it, just read it.

**Comment by DavidVorick** [comment]
Additionally again, you'll notice that you have not replaced all instances of 'v0.26.1' - they are still available in the filepaths.

Probably best to use this as your variable:

`electron-version = v0.xx.y`

**Comment by Mingling94** [comment]
Did it.

Plus I added a siacoin version variable in the same spirit.

File: Makefile [hunk 1]
```diff
@@ -13,23 +17,23 @@ run:
 	./node_modules/electron-prebuilt/dist/electron
 
 dist-linux-32:
-	wget -nc https://github.com/atom/electron/releases/download/v0.26.1/electron-v0.26.1-linux-ia32.zip
-	unzip -o electron-v0.26.1-linux-ia32.zip -d Sia-Linux-ia32
+	wget -nc https://github.com/atom/electron/releases/download/electron-version/electron-$(electron-version)-linux-ia32.zip
```
Comment (DavidVorick) [inline Makefile]: says 'electron-version' when it should say $(electron-version)

File: Makefile [hunk 2]
```diff
@@ -13,23 +17,23 @@ run:
...
+	unzip -o electron-$(electron-version)-linux-ia32.zip -d Sia-Linux-ia32
 	mv Sia-Linux-ia32/electron Sia-Linux-ia32/Sia
 	cp -R site/ Sia-Linux-ia32/resources/app/
-	tar -cJvf sia-v0.4.0-linux-ia32.tar.xz Sia-Linux-ia32
+	tar -cJvf sia-$(sia-version)-linux-ia32.tar.xz Sia-Linux-ia32
 
 dist-linux-64:
-	wget -nc https://github.com/atom/electron/releases/download/v0.26.1/electron-v0.26.1-linux-x64.zip
-	unzip -o electron-v0.26.1-linux-x64.zip -d Sia-Linux-x64
+	wget -nc https://github.com/atom/electron/releases/download/electron-version/electron-$(electron-version)-linux-x64.zip
```
Comment (DavidVorick) [inline Makefile]: ditto

File: Makefile [hunk 3]
```diff
@@ -13,23 +17,23 @@ run:
...
+	unzip -o electron-$(electron-version)-linux-x64.zip -d Sia-Linux-x64
 	mv Sia-Linux-x64/electron Sia-Linux-x64/Sia
 	cp -R site/ Sia-Linux-x64/resources/app/
-	tar -cJvf sia-v0.4.0-linux-x64.tar.xz Sia-Linux-x64
+	tar -cJvf sia-$(sia-version)-linux-x64.tar.xz Sia-Linux-x64
 
 dist-mac:
-	wget -nc https://github.com/atom/electron/releases/download/v0.26.1/electron-v0.26.1-darwin-x64.zip
+	wget -nc https://github.com/atom/electron/releases/download/electron-version/electron-$(electron-version)-darwin-x64.zip
```
Comment (DavidVorick) [inline Makefile]: ditto

File: Makefile [hunk 4]
```diff
@@ -41,21 +45,21 @@ dist-mac:
 	cp -R site/ Sia-Mac/Sia.app/Contents/Resources/app/
 	sed -i.bak s/Electron/Sia/g Sia-Mac/Sia.app/Contents/Info.plist
 	sed -i.bak s/Electron/Sia/g Sia-Mac/Sia.app/Contents/Frameworks/Sia\ Helper.app/Contents/Info.plist
-	tar -cJvf sia-v0.4.0-mac.tar.xz Sia-Mac
+	tar -cJvf sia-$(sia-version)-mac.tar.xz Sia-Mac
 
 dist-windows-32:
-	wget -nc https://github.com/atom/electron/releases/download/v0.26.1/electron-v0.26.1-win32-ia32.zip
-	unzip -o electron-v0.26.1-win32-ia32.zip -d Sia-Windows-ia32
+	wget -nc https://github.com/atom/electron/releases/download/electron-version/electron-$(electron-version)-win32-ia32.zip
```
Comment (DavidVorick) [inline Makefile]: ditto

File: Makefile [hunk 5]
```diff
@@ -41,21 +45,21 @@ dist-mac:
...
+	unzip -o electron-$(electron-version)-win32-ia32.zip -d Sia-Windows-ia32
 	mv Sia-Windows-ia32/electron.exe Sia-Windows-ia32/Sia.exe
 	cp -R site/ Sia-Windows-ia32/resources/app/
-	zip sia-v0.4.0-windows-ia32.zip Sia-Windows-ia32/*
+	zip sia-$(sia-version)-windows-ia32.zip Sia-Windows-ia32/*
 
 dist-windows-64:
-	wget -nc https://github.com/atom/electron/releases/download/v0.26.1/electron-v0.26.1-win32-x64.zip
-	unzip -o electron-v0.26.1-win32-x64.zip -d Sia-Windows-x64
+	wget -nc https://github.com/atom/electron/releases/download/electron-version/electron-$(electron-version)-win32-x64.zip
```
Comment (DavidVorick) [inline Makefile]: ditto

**Comment by Mingling94** [comment]
Good eye! Changed it

**Pull request merged**